src/python/aqueduct_executor/operators/system_metric_executor/main.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import base64
+import traceback
+import sys
+
+from aqueduct_executor.operators.utils import enums, utils
+from aqueduct_executor.operators.system_metric_executor import spec
+from aqueduct_executor.operators.utils.storage.parse import parse_storage
+
+
+def run(spec: spec.SystemMetricSpec) -> None:
+    """
+    Executes a system metric operator by storing the requested system metrics value in the output content path.
+    """
+    storage = parse_storage(spec.storage_config)
+    print(spec.metric_name)
```

Comment:
is this necessary? We already print the job spec.

Reply:
this was a mistake i printed it to confirm some testing thanks for catching!